ros/lib/utils.py

```diff
@@ -158,6 +158,11 @@ def calculate_percentage(numerator, denominator):
         return 0
 
 
+def systems_with_profile(org_id):
```

Comment:
```suggestion
def systems_ids_with_profile(org_id):
```

This method returns system ids and not system/profile records. Could you rename this method to something more specific?